features/partners.feature

```diff
@@ -139,74 +206,100 @@ Functionaliteit: Huwelijken en geregistreerd partnerschappen van een persoon raa
 
-    @gba
     Scenario: De partner heeft geslachtswijziging en naamswijziging ondergaan, maar de gevraagde persoon erkent dit niet
-      Gegeven de persoon met burgerservicenummer 555550001 heeft de volgende huwelijken/partnerschappen in de registratie
-      | Categorie | Burgerservicenummer (01.20) | Voornamen | geslacht (04.10) | Datum aangaan (06.10) | Datum ontbinding (07.10) |
-      | 5         | 555550002                   | Karel     | M                | 20091102              |                          |
-      En de partner met burgerservicenummer 555550002 heeft haar geslacht en voornamen gewijzigd zodat de volgende persoonsgegevens in de registratie op haar persoonslijst staan:
-      | Categorie | Voornamen (02.10) | Geslachtsaanduiding (04.10) |
-      | 1         | Charlotte         | V                           |
-      | 51        | Karel             | M                           |
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                        | waarde    |
+      | burgerservicenummer (01.20) | 555550001 |
+      En de persoon heeft een 'partner' met de volgende gegevens
+      | naam                                                                  | waarde    |
+      | burgerservicenummer (01.20)                                           | 555550002 |
+      | geslachtsaanduiding (04.10)                                           | M         |
+      | voornamen (05.02.10)                                                  | Karel     |
+      | datum huwelijkssluiting/aangaan geregistreerd partnerschap (05.06.10) | 20091102  |
+      En het systeem heeft een persoon met de volgende gegevens
+      | naam                        | waarde    |
+      | burgerservicenummer (01.20) | 555550002 |
+      | geslachtsaanduiding (04.10) | V         |
+      | voornamen (05.02.10)        | Charlotte |
       Als personen wordt gezocht met de volgende parameters
       | naam                | waarde                                               |
       | type                | RaadpleegMetBurgerservicenummer                      |
       | burgerservicenummer | 555550001                                            |
       | fields              | partners.burgerservicenummer,partners.naam.voornamen |
-      Dan heeft de persoon met burgerservicenummer '555550001' alleen 'partners' met de volgende gegevens
+      Dan heeft de response een persoon met een 'partner' met de volgende gegevens
       | naam                | waarde    |
       | burgerservicenummer | 555550002 |
-      | geslacht            | M         |
-      En heeft de partner met burgerservicenummer '555550002' de volgende 'naam' gegevens
+      En heeft de 'partner' de volgende 'naam' gegevens
       | naam      | waarde |
       | voornamen | Karel  |
 
-
-  @proxy
   Rule: Als er een actueel huwelijk of geregistreerd partnerschap is wordt alleen dit actueel huwelijk of geregistreerd partnerschap teruggegeven,
```

Comment:
Het lijkt er op alsof in deze Rule de in regel 14 beschreven Rule nog een keer (deels) overnieuw wordt gedaan.
Welicht beide Rules samenvoegen en natuurlijk ook de scenario's dan meenemen.

Reply:
op regel 14 staat de gba rule, op regel 300 de proxy rule. lijkt inderdaad dubbelop